### PR TITLE
CONFIGURE: Fix incorrect help text

### PR DESCRIPTION
**Pull request by raziel-**
On using the path where libcurl is installed (assuming that the old help text is referring to the actual library, be it static or shared) this will result in libcurl not being found by configure.
Instead, the path should point to where libcurl-config is installed, so it may get picked up and processed properly
(See also --with-freetype-prefix and --with-sdl-prefix)
e.g.
On my SDK installaliton libcurl.a and libcurl.so are placed in /sdk/local/newlib/lib, while libcurl-config is in /sdk/local/newlib/bin.
Using the latter makes configure properly picking up and working with libcurl. The other don´t.

Thank you for contributing to ScummVM. Please read the following carefully before submitting your Pull Request.

Make sure your individual commits follow the guidelines found in the ScummVM Wiki: https://wiki.scummvm.org/index.php?title=Commit_Guidelines. If they're not please edit them before submitting the Pull Request.

Commits and Pull Requests should use the following template:

```
SUBSYSTEM: Short (50 chars or less) summary of changes

More detailed explanatory text, if necessary.  Wrap it to about 72
characters or so.  In some contexts, the first line is treated as the
subject of an email and the rest of the text as the body.  The blank
line separating the summary from the body is critical (unless you omit
the body entirely); tools like rebase can get confused if you run the
two together.

Write your commit message in the present tense: "Fix bug" and not "Fixed
bug."  This convention matches up with commit messages generated by
commands like git merge and git revert.

Further paragraphs come after blank lines.

- Bullet points are okay, too

- Typically a hyphen or asterisk is used for the bullet, preceded by a
 single space, with blank lines in between, but conventions vary here

- Use a hanging indent
```

### CODE REVIEW

**Comment by lotharsm** [comment]
One thing I missed on the last PR:

Please alter the commit message according to our guidelines:

CONFIGURE: Fix incorrect help text

File: configure [hunk 1]
```diff
@@ -4876,7 +4876,7 @@ echo "$_libunity"
 #
 find_freetype() {
         # Wrapper function which tries to find freetype
-        # either by calling freetype-config or by using
+        # either by callimg freetype-config or by using
```
Comment (criezy) [inline configure]: Why that change? Wasn’t `calling` correct?

Comment (raziel-) [reply configure]: Grr, of course it was correct.
I thought i was adding this change to my other patch while it was being merged and ended up creating a new PR but with the old data.
Sorry

**Comment by criezy** [comment]
Usually `prefix` in this context means one level up, so in your case `/sdk/local/newlib`. Have you tried to use that? It should work.

So I think the old text is actually correct, as it is not asking for the lib directory, but for this `prefix.` And I would tend to reject your change.

**Comment by raziel-** [comment]
Heh, i'm so noob that it hurts.

Yes, that works well too.

Sorry for the noise :-/

**Pull request closed without merging**